compiler/src/iree/compiler/Codegen/Passes.td

```diff
@@ -635,6 +635,14 @@ def LLVMCPULowerExecutableTarget :
       "mlir::iree_compiler::createLLVMCPULowerExecutableTargetPass()";
 }
 
+def ExpandF16MaxFToF32 :
```

Comment:
I assume that there could be more arith ops needing the conversion? If so, we can rename it to `ExpandArithF16ToF32`, so we don't only scope it to max op.